Knowledge-Base/Windows-arm64-vm-using-qemu.md

```diff
@@ -0,0 +1,113 @@
+# Windows 11 VM installation on ARM64 with qemu
+
+This guide explains how to install and run a Windows 11 for ARM64 virtual machine with `qemu-system-aarch64`.
+
+## Prerequisites
+
+### Windows ISO (ARM64)
+- Download the Windows 11 (ARM64) ISO from Microsoft’s download page: [Windows 11 for Arm-based PCs](https://www.microsoft.com/en-us/software-download/windows11arm64).
+
+### Virtio drivers
+- Download the latest `virtio-win` driver ISO from the repository page: [virtio-win](https://github.com/virtio-win/virtio-win-pkg-scripts/blob/master/README.md). It will be used during setup to provide the storage and network drivers.
+
+### EFI
+- Install the AArch64 UEFI firmware from your distribution (packages: `qemu-efi-aarch64` or `edk2-aarch64`). This provides `QEMU_EFI.fd`.
+- If it is not available in your distro, download the Debian package: [qemu-efi-aarch64](https://packages.debian.org/sid/qemu-efi-aarch64) and extract it using:
+
+```bash
+ar x qemu-efi-aarch64*.deb
+tar xvf data.tar.xz
+cp ./usr/share/qemu-efi-aarch64/QEMU_EFI.fd .
+```
+
+### Create a virtual disk
+```bash
+qemu-img create win11-arm64.img 100G
+```
+
+You should now have the following files in your working directory:
+- `Win11_*.iso`
+- `virtio-win-*.iso`
+- `QEMU_EFI.fd`
+- `win11-arm64.img`
+
+## Run QEMU
+
+### Option A: Software emulation (TCG)
+
+```bash
+qemu-system-aarch64 \
+  -M virt -m 8G -cpu max,pauth-impdef=on -smp 8 \
+  -bios ./QEMU_EFI.fd\
+  -accel tcg,thread=multi\
+  -device ramfb \
+  -device qemu-xhci -device usb-kbd -device usb-tablet \
+  -nic user,model=virtio-net-pci \
+  -device usb-storage,drive=install \
+  -drive if=none,id=install,format=raw,media=cdrom,file=./Win11_*.iso \
+  -device usb-storage,drive=virtio-drivers \
+  -drive if=none,id=virtio-drivers,format=raw,media=cdrom,file=./virtio-win-*.iso \
+  -drive if=virtio,id=system,format=raw,file=./win11-arm64.img  
+```
+
+### Option B: Hardware acceleration (KVM, aarch64 host)
+
+Requires a Linux aarch64 host with KVM enabled. On x86_64 hosts, use Option A.
+From the Option A command, change:
+- `-accel tcg,thread=multi` -> `-accel kvm`
+- `-cpu max,pauth-impdef=on` -> `-cpu host`
+Everything else remains the same.
+
+
+## Windows setup walkthrough
+
+1. Boot from the Windows ISO
+   - When prompted, press any key to boot from the installer.
+2. Start installation
+   - Choose language, click Install. When asked for a key, click “I don’t have a product key”.
+   ![Windows setup: language and Install](https://github.com/user-attachments/assets/612667af-ecc9-4c8a-9157-dabeb18e541f)
+   
+3. Bypass Windows 11 checks (if shown)
+   - If you see “This PC can’t run Windows 11”, press Shift+F10 → run `regedit`.
+   - Navigate to `HKEY_LOCAL_MACHINE\SYSTEM\Setup`.
+   - Right-click `Setup` → New → Key → name it `LabConfig`.
+   - Right-click `LabConfig` → New → DWORD (32-bit) Value → create:
+     - `BypassCPUCheck`, `BypassRAMCheck`, `BypassSecureBootCheck`, `BypassStorageCheck`, `BypassTPMCheck`
```

Comment:
@elizashurov hi, a minor suggestion, currently it seems that only securebootcheck need to bypass.

Reply:
Might be useful sometimes to install without TPM. So maybe separate this section for "must" and "can be done for convenience".